addOns/retire/CHANGELOG.md

```diff
@@ -4,7 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+### Added
+- Added helpful description for retire.js add-on
```

Comment:
Drop "added" it's the heading and just start with "A helpful...". Finishing with a period would be good too.